Documentation/concepts/observability/hubble-configuration.rst

```diff
@@ -127,69 +137,61 @@ it has the following advantages:
 
 **Troubleshooting**:
 
-If you get the following error while install cilium (and cert-manager), it's because
-cert-manager ValidatingWebhook (which used to verify the Certificate CRD resources)
-is not available (blocked due to CNI is not available).
+While installing Cilium or cert-manager you may get the following error:
 
 ::
 
     Error: Internal error occurred: failed calling webhook "webhook.cert-manager.io": Post "https://cert-manager-webhook.cert-manager.svc:443/mutate?timeout=10s": dial tcp x.x.x.x:443: connect: connection refused
 
-There are several ways to overcome above issue, such as:
+This happen when cert-manager's webhook (used to verify the ``Certificate`` CRD
+resources) is not available. There are several ways to workaround this issue:
```

Comment:
nit, visually users will get the hint but we can be a little bit more explicit here about what the user should do (in particular, do one, but don't do all). Also are these workarounds or are they they actual solution to this problem?

```suggestion
resources) is not available. There are several ways to resolve this issue.
Pick one of the options below:
```